ibm/resource_ibm_is_vpc_routing_table_route.go

```diff
@@ -63,11 +64,17 @@ func resourceIBMISVPCRoutingTableRoute() *schema.Resource {
 				ForceNew:    true,
 				Description: "The zone to apply the route to. Traffic from subnets in this zone will be subject to this route.",
 			},
-			rNextHop: {
```

Comment:
Don't change this rNextHop just by introducing two different variables...This will break all existing cutomers

Can't we go with same name and do the same logic cli is having and set the respective value?